framework/database/__init__.py

```diff
@@ -105,7 +105,7 @@ def paginated(model, query=None, increment=200, each=True, include=None):
         are yielded.
     """
     if include and query:
-        queryset = model.objects.filter(query).include(*include)
+        queryset = model.objects.filter(query).prefetch_related(*include)
```

Comment:
Probably unrelated but `def paginated(model, query=None, increment=200, each=True, include=None):` has an undocumented `include` param. Do you know what it is? I wasn't able to find usages of this param in the code base.